azure/table_azure_storage_account_local_user.go

```diff
@@ -0,0 +1,248 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/storage/armstorage"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureStorageAccountLocalUser(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_storage_account_local_user",
+		Description: "Azure Storage Account Local User",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"storage_account_name", "name", "resource_group"}),
+			Hydrate:    getStorageAccountLocalUser,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound", "404"}),
+			},
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listStorageAccounts,
+			Hydrate:       listStorageAccountLocalUsers,
```

Comment:
Can we add an optional key qualifier to skip the API call when `storage_account_name` is provided as a query parameter?